app/models/Organisation.js

```diff
@@ -47,7 +47,7 @@ const orgSchema = new Schema({
     }
   },
   logo: {
-    type: Buffer,
+    image: Buffer,
```

Comment:
add one more field here imgUrl, look in other collections what we have named it and use the same name. this is for saving the aws link of the img